api/agent/config.go

```diff
@@ -58,6 +59,8 @@ const (
 	EnvDockerNetworks = "FN_DOCKER_NETWORKS"
 	// EnvDockerLoadFile is a file location for a file that contains a tarball of a docker image to load on startup
 	EnvDockerLoadFile = "FN_DOCKER_LOAD_FILE"
+	// EnvDockerDisableSecurity disables docker security features like user name, cap drop etc.
```

Comment:
Nit: is the intent that this will also be a switch for other security features that are eventually integrated?  If not, it might be worthwhile to pick a more precise and descriptive name.

Reply:
yeah, good question, we currently set docker's User, CapDrop and SecurityOpt, so I couldn't find more descriptive name, did you have some in mind?

Reply:
Not sure that I have any brilliant suggestions.  Does `FN_DISABLE_UNPRIVILEGED_CONTAINERS` sound more descriptive, or just more confusing?

Reply:
sounds good to me :-)
one possible inconvenience: docker has `--privileged` option and we don't use it with this setting, so it might be little confusing...